### PR TITLE
Fix multiphase unit tests to run on GPUs

### PR DESCRIPTION
**Pull request by mbkuhn**
addresses issue #964

### CODE REVIEW

**Comment by mbkuhn** [comment]
Note: all unit tests passed on GPUs on Eagle using this branch

File: unit_tests/multiphase/test_momflux.cpp [hunk 1]
```diff
@@ -291,10 +295,44 @@ class MassMomFluxOpTest : public MeshTest
...
+        constexpr amrex::Real w_check = 0.0;
+        constexpr amrex::Real advrho_check = 0.0;
+        constexpr amrex::Real dqdt_check = 0.0;
+
+        // Check error in each mfab
+        for (int lev = 0; lev < repo.num_active_levels(); ++lev) {
+            // Sum error for each component individually
+            amrex::Real err_lev = error_fld(lev).max(0);
+            // Check error
+            EXPECT_NEAR(err_lev, umac_check, tol);
```
Comment (marchdf) [inline unit_tests/multiphase/test_momflux.cpp:320]: want to just make all these 1 liners? `EXPECT_NEAR(error_fld(lev).max(0), umac_check, tol);`

File: unit_tests/multiphase/test_vof_cons.cpp [hunk 2]
```diff
@@ -253,7 +253,25 @@ class VOFConsTest : public MeshTest
...
+            get_accuracy(error_fld, dir, m_nx, vof);
+
+            // Check error in each mfab
+            constexpr amrex::Real vofsol_check = 0.0;
+            for (int lev = 0; lev < repo.num_active_levels(); ++lev) {
+                // Sum error for each component individually
+                amrex::Real err_lev = error_fld(lev).max(0);
+                // Check error
+                EXPECT_NEAR(err_lev, vofsol_check, tol);
+                err_lev = error_fld(lev).max(1);
```
Comment (marchdf) [inline unit_tests/multiphase/test_vof_cons.cpp:273]: missing an expect near here? also could make these checks (and some a bit above in vof) one-liners

File: unit_tests/multiphase/test_momflux.cpp (hunk 1 above)
Comment (mbkuhn) [reply unit_tests/multiphase/test_momflux.cpp:320]: great idea. I'll do that.

File: unit_tests/multiphase/test_vof_cons.cpp (hunk 2 above)
Comment (mbkuhn) [reply unit_tests/multiphase/test_vof_cons.cpp:273]: unintended max - I only have one component of error en this test. Maybe this is why the unit tests fail in the CI?

**Review by marchdf — APPROVED** [review]

**Comment by mbkuhn** [comment]
I think the previously failing unit test was either from calling max() on a field component that didn't exist or from putting data in a ghost cell of the error scratch field that didn't exist.

**Pull request merged**